src/ui/components/ShowMoreCard/index.js

```diff
@@ -54,7 +56,11 @@ export class ShowMoreCardBase extends React.Component {
         </div>
         <a className="ShowMoreCard-revealMoreLink" href="#show-more"
           onClick={this.expandText}>
-          {i18n.gettext('Show more info…')}
+          <span className="visually-hidden">
```

Comment:
This might not translate well if we want to flip the two strings in a certain language. It might need to be `i18n.gettext('<span class="visually-hidden">Expand to</span> Read More')`. Putting HTML in our translations is definitely not ideal though.

There should at least be an l10n comment explaining how these strings are used.

It would be nice if we could get the text formatted somehow but that could be confusing. Something like this, but actually reasonable:

```jsx
const { label, text } = magicallyGetI18nValues(
  i18n.gettext('label(Expand to) text(Read More)'));
// ...
  <span className="visually-hidden">{label.trim()}</span>{text.trim()}